pynailgun/ng.py

```diff
@@ -1029,19 +1029,19 @@ def main():
 
     # The command we need to execute gets written
     # to the --out-file parameter. If the user
-    # hasn't specified this we hijack it 
+    # hasn't specified this we hijack it
     out_file_temp = tempfile.NamedTemporaryFile()
     out_file_path = out_file_temp.name
-    if cmd == "console": 
+    if cmd == "console":
         try:
             index = cmd_args.index("--out-file")
-        except: 
+        except:
             index = -1
 
         if index != -1:
-            out_file_path =  cmd_args[index + 1]
+            out_file_path = cmd_args[index + 1]
         else :
-            cmd_args = [cmd_args[0], "--out-file", out_file_path] + cmd_args[1:]
+            cmd_args = cmd_args + ["--out-file", out_file_path]
```

Comment:
wouldn't this result in repeated args? given the origin of this PR, maybe `bloop console` should just lead to an error  like "pls specify project"?

Reply:
This doesn’t lead to repeated args, because this branch is only reached if the user has not specified the `—out-file` arg.

With this fix, `bloop console` with no args calls the underlying bloop client, which returns a useful error message along the lines of “required project name missing”, so the behaviour is the same as calling `bloop compile` with no args.